README.md

```diff
@@ -30,8 +30,37 @@ To add or update reference images, edit the scenarios in `backstop.json` and run
 
 ### Translation Development
 
+DIMES is currently translated into the following languages:
+  * Chinese (Simplified)
+  * French
+  * German
+  * Italian
+  * Japanese
+  * Korean
+  * Portuguese (Brazilian)
+  * Spanish
+  * Turkish
+
 The repository includes [linguijs](https://lingui.dev/) which is an [Open-source](https://github.com/lingui/js-lingui) Internationalization Framework.  This allows you to do translation work on the UI.
 
+Lingui requires a configuration file (located in the base directory and named `lingui.config.js`). See the [official documentation](https://lingui.dev/ref/conf) for full instructions on configuration.
+
+DIMES uses a [macro implementation](https://lingui.dev/guides/message-extraction#macro-usages) for message extraction. This means that all text with a `<Trans></Trans>` block will be automatically extracted for localization files.
+
+You must update the extractions any time new translation objects or languages are found:
```

Comment:
```suggestion
You must update files used by lingui any time new translation objects or languages are found:
```

Reply:
Wondering if we should break this section up into three different processes:
- Adding a new language
- Updating HTML structure
- Changing translation strings

The commands needed in each case vary, and it might be helpful to have that laid out a bit more clearly. Thoughts?

Reply:
Looks good! I think we can get rid of that "You must update files used by lingui any time new translation objects or languages are found:" text now because it's covered by the sections below.